docs/etcd.md

```diff
@@ -0,0 +1,84 @@
+# Experimental etcd support in LND
+
+With the recent introduction of the `kvdb` interface LND can support multiple
+database backends allowing experimentation with the storage model as well as
+improving robustness trough eg. replicating essential data.
+
+Building on `kvdb` in v0.11.0 we're adding experimental [etcd](https://etcd.io)
+support to LND. As this is an unstable feature heavily in development, it still
+has *many* rough edges for the time being. It is therefore highly recommended to
+not use LND on `etcd` in any kind of production environment especially not
+on bitcoin mainnet.
+
+## Building LND with etcd support
+
+To create a dev build of LND with etcd support use the following command:
+
+```
+make tags="autopilotrpc signrpc walletrpc chainrpc invoicesrpc watchtowerrpc kvdb_etcd"
```

Comment:
the other build tags seem irrelevant here?

Reply:
agreed, removed